global/nix-modules/5/flake.nix

```diff
@@ -0,0 +1,44 @@
+{
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+    # ⤵️ flake4 is specified as input
+    flake4.url = "path:../4";
+  };
+  outputs = { self, nixpkgs, flake4 }:
+    let
+      # TODO: Change this to x86_64-linux if you are on Linux
+      system = "aarch64-darwin";
+      pkgs = nixpkgs.legacyPackages.${system};
+      lib = pkgs.lib;
+      lsdFor = settings:
```

Comment:
Perhaps it'd be good to put this in a flake output attribute like `lib.lsdFor`, and use `self.lib.lsdFor` in the example packages.

Reply:
Good idea, though it will have to be `mkLib` since we need `pkgs`.